src/condor_startd.V6/Starter.cpp

```diff
@@ -524,13 +524,13 @@ Starter::exited(Claim * claim, int status) // Claim may be NULL.
 		dummyAd.Assign(ATTR_IMAGE_SIZE, 0);
 		dummyAd.Assign(ATTR_JOB_CMD, "boinc");
 		std::string gjid;
-		formatstr(gjid,"%s#%d#%d#%d", get_local_hostname().c_str(), now, 1, now);
+		formatstr(gjid,"%s#%lld#%d#%lld", get_local_hostname().c_str(), (long long)now, 1, (long long)(long long)now);
```

Comment:
```suggestion
		formatstr(gjid,"%s#%lld#%d#%lld", get_local_hostname().c_str(), (long long)now, 1, (long long)now);
```
Excessive double casting to `long long`